fecha.js

```diff
@@ -188,7 +188,8 @@
         d.isPm = true;
       }
     }],
-    ZZ: [/[\+\-]\d\d:?\d\d/, function (d, v) {
+    ZZ: [/([\+\-]\d\d:?\d\d|Z)/, function (d, v) {
```

Comment:
I think we should maybe add a non-capturing group to separate the "digits" case from the "Z". The way it’s written it accepts a string like this: "+09:3Z"